packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -493,4 +496,12 @@ export abstract class SharedObject<TEvent extends ISharedObjectEvents = ISharedO
     private reSubmit(content: any, localOpMetadata: unknown) {
         this.reSubmitCore(content, localOpMetadata);
     }
+
+    private rebaseOp(content: any, localOpMetadata: unknown) {
+        this.rebaseOpCore(content, localOpMetadata);
+    }
+
+    protected rebaseOpCore(content: any, localOpMetadata: unknown) {
```

Comment:
I think it has to be abstract function. Having it empty is dangerous, as DDS implementers may forget to implement it